app/lib/style.ts

```diff
@@ -67,14 +67,15 @@ export const colorTheme = (
   const text = darkMode ? DARK_MODE_WHITE : 'black';
   const secondary = darkMode ? '#505050' : '#ccc';
   const error = mix(adjustHue(p, 280), 'white', darkMode ? 0.3 : 0);
+  const lighter = mix(p, cellBG, 0.6);
   return {
     '--tag-l': darkMode ? '30%' : '85%',
     '--bg': bg,
     '--primary': p,
     '--blue': darkMode ? mix('blue', 'white', 0.5) : 'blue',
     '--green': darkMode ? mix('green', 'white', 0.5) : 'green',
     '--onprimary': readableColor(p, darkMode),
-    '--lighter': mix(p, cellBG, 0.6),
+    '--lighter': lighter,
```

Comment:
similar to the `entryCell` comment above - I'm OK with this but have an ever so slight pref for undoing the change since it doesn't seem to be a necessary part of the PR.

Reply:
ah right, I missed those changes when I reviewed but knew I was forgetting something.